docs/index.rst

```diff
@@ -50,7 +50,8 @@ Mission Statement
 
     installation
     quickstart/quickstart
-
+    tutorials
+    API <api/modules>
```

Comment:
I like the old API subsection, but let see what others think about it!

Reply:
Ok, completely understandable. Just to put it out there, there are a few reasons I did that:
- It shortens the sidebar, which is an overall goal of mine in making the documentation easier to navigate.
- It makes the API easier to find. As we continue fixing up docstrings, I think the API should be closer to the top than it was before. I think this way puts it in a place where it makes sense.
- The single page in its own section looked sloppy IMO (I have plans for most of the other sections like that).

Reply:
I think these reasons do make sense - esp making API easier to find also for new users